src/lib/protocols/credentials/__tests__/StubWallet.ts

```diff
@@ -4,6 +4,12 @@ import { UnpackedMessageContext } from '../../../types';
 
 export class StubWallet implements Wallet {
   private wh?: number | undefined;
+
+  public get walletHandle() {
```

Comment:
Could we keep it private?

Reply:
I see the `SignatureDecoratorUtils.test.ts` uses it:

```ts
const verkey = await indy.createKey(wallet.walletHandle as number, { seed: seed1 });
```

The test compares the result from the wallet to a native indy call. So this woud need a refactor. It was already public. Or do you mean just the `StubWallet`?

Reply:
I left `walletHandle` in `IndyWalet` public for now. but applied the rest of your feedback